### DISCUSSION TITLE
Dynamic properties in Templater don't work

### DISCUSSION THREAD

**Discussion by ETforBeginners** [SilentVoid13/Templater#1281] [Help]
Again and again: I have the problem with the use of frontmatter properties. I use for instance <%+ tp.frontmatter.thema%>  in my notes. 

The easiest test:

--------------------------------------
---
thema: Test
---
<%+ tp.frontmatter.thema%>
--------------------------------------

leads to:

--------------------------------------
undefined
--------------------------------------

ore insert with a template to:

--------------------------------------
NaN
--------------------------------------
Any ideas? I tried it in a new Vault as  well...
Thanks a lot!

OS: Windows 10 home 
Obsidian: 1.5.3
templater: 2.0.0

**Comment by Zachatoo** [comment]
I'd recommend using Dataview for dynamic content instead of Templater. Dataview consistently is better at that, so much so that we're planning on deprecating the dynamic feature in Templater (see #913 ).

**Comment by ETforBeginners** [comment]
Thank you for the quick answer and the clear words. It seems to me to be a good idea to add this as a note in the help of Templater. Kind regards
